jobio-back/backend.js

```diff
@@ -0,0 +1,126 @@
+const { formatDate } = require('./utils/functions');
+const { custom_queries } = require('./utils/queries');
+
+const { pool } = require('./db/db');
+const express = require('express');
+const cors = require('cors');
+
+const app = express();
+const PORT = 3001;
+
+app.use(express.json());
+app.use(cors({ origin: "*" }));
+
+app.listen(PORT, () => {
+    console.log(`the server is listening to the port ${PORT}`)
+})
+
+pool.query('SELECT NOW()', (err, res) => {
+    if (err) {
+        console.error('Error connecting to the database:', err);
+    } else {
+        console.log('Connected to the database on', res.rows[0].now);
+    }
+});
+
+app.get('/getOrgsNames', (req, res) => {
+    pool.query(custom_queries.getOrgNames)
+        .then(pooledData => { res.status(200).json(pooledData) })
+})
+
+app.get('/getSeekersNames', (req, res) => {
+    pool.query(custom_queries.getSeekersNames)
+        .then(pooledData => { res.status(200).json(pooledData) })
+})
+
+app.get('/getJobPostsNames', (req, res) => {
+    pool.query(custom_queries.getJobPostsNames)
+        .then(pooledData => { res.status(200).json(pooledData) })
+})
+
+app.post('/newUserSignUp', (req, res) => {
+    const body = req.body;
+    let user_role;
+    body.role === 'Organization' ? user_role = 'org' : user_role = 'seeker';
+    const currentDate = new Date();
+    const formattedDate = formatDate(currentDate);
+
+    pool.query(custom_queries.addNewUserToUniqueUserId, [body.uid, formattedDate, user_role])
+    user_role === 'seeker'
```

Comment:
you should try to avoid hard coded strings in your app and instead have a `constants.js` file where you can import your constants from. This way if you need to change one constant, you can do it in one place.